bin/fin

```diff
@@ -3376,6 +3381,7 @@ project_create ()
 	echo -e "Project folder:   ${yellow}$(pwd)/$project_name${NC}"
 	echo -e "Project software: ${yellow}${target_cms}${NC}"
```

Comment:
```suggestion
	echo -e "Project software: ${yellow}${target_cms:-<unknown>}${NC}"
```

Target CMS can be unknown now

Reply:
@achekulaev `target_cms="Custom git repository"` in the case:

```
	# Read repo from params if available
	if [[ "$repo" != "" ]]; then
		local target_repo="$repo"
		local choice="0"
	fi
```